developer-docs-site/docs/concepts/move.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Move - A Web3 Language and Runtime"
+slug: "move-on-aptos"
+---
+
+# Move - A Web3 Language and Runtime
+
+The Aptos blockchain consists of validator nodes that run a consensus protocol. The consensus protocol agrees upon the ordering of transactions and their output when executed on the Move Virtual Machine (MoveVM). Each validator node translates transactions along with the current blockchain ledger state as input into the VM. The MoveVM processes this input to produce a changeset or storage delta as output. Once consensus agrees and commits to the output, it becomes publicly visible. In this guide, we will introduce you to core Move concepts and how they apply to developing on Aptos.
+
+## What is Move?
+
+Move is a safe and secure programming language for Web3 that emphasizes **scarcity** and **access control**. Any assets in Move can be represented by or stored within *resource*. **Scarcity** is enforced by default as structs cannot be accidentally duplicated or dropped. Only structs that have explicitly been defined at the bytecode layer as *copy* can be duplicated and *drop* can be dropped, respectively.
+
+**Access control** comes from both the notion of accounts as well as module access privileges. A module in Move may either be a library or a program that can create, store, or transfer assets. Move ensures that only public module functions may be accessed by other modules. Unless a struct has a public constructor, it can only be constructed within the module that defines it. Similarly, fields within a struct can only be accessed and mutated within its module that or via public accessors and setters. Furthermore, structs defined with *key* can be stored and read from global storage only within the module defines it. Structs with *store* can be stored within another *store* or *key* struct inside or outside the module that defines that struct.
+
+In Move, a transaction's sender is represented by a *signer*, a verified owner of a specific account. The signer has the highest level of permission in Move and is the only entity capable of adding resources into an account. In addition, a module developer can require that a signer be present to access resources or modify assets stored within an account.
+
+## Comparison to other VMs
```

Comment:
Note: I know these are old and legacy docs, but leaving comments nevertheless so we do not forget to change in the future, plus since you are touching these files ...😂

Reply:
Yeah, I don't mind as long as we don't spend too much time on getting this right. This shouldn't be seen as a definitive and final edit. I'd like for us to be comfortable iterating frequently in docs. Although I recognize there's always tension in doing anything....

Reply:
Sure, I think this is better to fix separately, sadly we only see such things when someone refactors / rearranges things :(